crates/nri-resctrl-plugin/src/lib.rs

```diff
@@ -169,10 +351,13 @@ impl<P: FsProvider + Send + Sync + 'static> Plugin for ResctrlPlugin<P> {
             req.runtime_name, req.runtime_version
         );
 
-        // Subscribe to minimal container and pod lifecycle events.
+        // Subscribe to container and pod lifecycle events we handle.
         let mut events = EventMask::new();
         events.set(&[
             Event::CREATE_CONTAINER,
+            Event::UPDATE_CONTAINER,
+            Event::STOP_CONTAINER,
```

Comment:
why do we need UPDATE_CONTAINER? CREATE_CONTAINER should be sufficient for us.

why do we need STOP_CONTAINER? REMOVE_CONTAINER should be sufficient?